src/hotspot/share/prims/jvm.cpp

```diff
@@ -2976,11 +2976,6 @@ JVM_ENTRY(void, JVM_StopThread(JNIEnv* env, jobject jthread, jobject throwable))
 JVM_END
 
 
-JVM_ENTRY(jboolean, JVM_IsThreadAlive(JNIEnv* env, jobject jthread))
-  oop thread_oop = JNIHandles::resolve_non_null(jthread);
-  return java_lang_Thread::is_alive(thread_oop);
-JVM_END
-
 
```

Comment:
Seems like we should better remove one empty line more (like original change).

Reply:
Right! Fixed in new commit.